mod_reforged/hooks/config/strings.nut

```diff
@@ -219,7 +219,7 @@ local vanillaDescriptions = [
 		ID = "perk.bullseye",
 		Key = "Bullseye",
 		Description = ::UPD.getDescription({
-			Fluff = "Nailed it!",
+			Fluff = "One in the hand is worth two in the bush.",
```

Comment:
How about? `One through the gaps is worth two/many in the armor.`

Reply:
I like 'one in the hand' better